### PR TITLE
Create MetaMaint.py

### PR DESCRIPTION
**Pull request by themousepotato**
TODO:
* Add Random_package template to sidebar of homepage(which I don't have access)
* Deploy and make it run once in a day.

### CODE REVIEW

File: jobs/pywikibot/scripts/MetaMaint.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import pywikibot
+from random import randint
+from datetime import date
+
+def main():
+    site = pywikibot.Site()
+    
+    # Read package blurbs
+    rpage = pywikibot.Page(site, 'Package_blurbs')
```
Comment (amrav) [inline jobs/pywikibot/scripts/MetaMaint.py]: Nit: probably better to put this under the `Metakgp` namespace, like `Metakgp:Package_blurbs`, so it doesn't show up in recent pages, trending, etc.

**Review by amrav — APPROVED** [review]
Great, thanks for doing this! You could add the cron config into this PR too if you like.

**Review by icyflame — APPROVED** [review]
I made a test page to see how this would look [here](https://wiki.metakgp.org/w/User:Icyflame/main_page_testing)

**Comment by icyflame** [comment]
Will merge this PR tonight, if there are no other comments. (We need to edit the Main page HTML to put this template somewhere)

cc @amrav

**Review by icyflame — CHANGES_REQUESTED** [review]
@TheMousePotato You still need to add the bash script that will run this script, and the line in the crontab. I think we should merge this once we have both of those things.

**Comment by icyflame** [comment]
Preview:

image

**Comment by amrav** [comment]
Why not feature it on the main column? The sidebar is getting too busy, we shouldn't put more than three things in there. Would be great to add screenshots as well.

Once a day also seems too frequent for updates, perhaps once every three days or once a week will allow each featured project to get more visibility.

Improvement for a future PR: can we do better than random selection, perhaps weighting projects by [frecency](https://en.wikipedia.org/wiki/Frecency) of commit activity?

**Comment by amrav** [comment]
Forgot to ask, @TheMousePotato could you attach a log of a successful run, to show that it actually works?

**Comment by icyflame** [comment]
Hey @TheMousePotato I have closed #41 and #42. Please make the changes inside this PR itself.

**Comment by themousepotato** [comment]
screenshot from 2018-03-31 00-16-42

@amrav You were talking about this template, right?

**Comment by icyflame** [comment]
Hey @TheMousePotato can you add all the files in this PR so that we can go ahead and merge this?

**Comment by themousepotato** [comment]
@icyflame Added all files.

**Review by icyflame — APPROVED** [review]

**Pull request merged**